### ISSUE TITLE
Unable to work with KeyVault certificates from azure-cli

### ISSUE THREAD

**Issue by ehonsey** [Azure/azure-cli#15686]
## Describe the bug

**Command Name**
`az keyvault certificate list`

**Errors:**
```
No module named 'azure.keyvault.v7_0'
Traceback (most recent call last):
python3/dist-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 528, in execute
    self.commands_loader.load_arguments(command)
azure/cli/core/__init__.py, ln 299, in load_arguments
    self.command_table[command].load_arguments()  # this loads the arguments via reflection
cli/core/commands/__init__.py, ln 291, in load_arguments
    super(AzCliCommand, self).load_arguments()
...
<frozen importlib._bootstrap>, ln 961, in _find_and_load_unlocked
<frozen importlib._bootstrap>, ln 219, in _call_with_frames_removed
<frozen importlib._bootstrap>, ln 1014, in _gcd_import
<frozen importlib._bootstrap>, ln 991, in _find_and_load
<frozen importlib._bootstrap>, ln 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'azure.keyvault.v7_0'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az keyvault certificate list`

## Expected Behavior

Receive a list of certificates in the keyvault. Not sure why azure-cli cannot work with certificates in the KeyVault for app services.

## Environment Summary
```
Linux-4.19.128-microsoft-standard-x86_64-with-glibc2.29
Python 3.8.2
Shell: zsh

azure-cli 2.0.81

Extensions:
azure-devops 0.17.0

```
## Additional Context

Ran command with azure-cli in WSL Ubuntu 20.04 distribution

**Comment by yungezz** [comment]
hi @bim-msft could you pls have a look? thanks

**Comment by ehonsey** [comment]
The `az appservice` command was giving similar issues. I realized that I took the `apt` install path for WSL (over Ubuntu 20.04). I removed that version of azure-cli and reinstalled with the one command path from here: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt. 

Now the same issues with Python are not resurfacing. Maybe this indicates that the manual install path with `apt` is broken? Or some dependency was missing on that install.
